### PR TITLE
add code to list author of top repos

### PR DESCRIPTION
**Pull request by wemeya**

### CODE REVIEW

File: dao/accesslog.go [hunk 1]
```diff
@@ -159,5 +159,21 @@ func GetTopRepos(countNum int) ([]models.TopRepo, error) {
 	if err != nil {
 		return nil, err
 	}
+	var repoString string
+	for _, v := range lists {
```
Comment (reasonerjt) [inline dao/accesslog.go]: Why use plural as it's only one list?

File: dao/accesslog.go [hunk 2]
```diff
@@ -159,5 +159,21 @@ func GetTopRepos(countNum int) ([]models.TopRepo, error) {
...
+		repoString += v.RepoName + ","
+	}
+	repoString = repoString[0 : len(repoString)-1]
```
Comment (reasonerjt) [inline dao/accesslog.go]: This will throw runtime error if the list is empty.

**Comment by reasonerjt** [comment]
I may have missed some discussion but there's a question, different tags of a repository may be pushed by different users, so why add "creator" as attribute of a repository?

**Comment by wemeya** [comment]
@reasonerjt added according to the new UI

**Pull request closed without merging**